src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -394,8 +404,8 @@ def mk_dirs(self, fs_id, path):
         cfs = self._cephfs_instance(fs_id)
         cfs.mk_dirs(path)
 
-    @RESTController.Resource('POST')
-    def rm_dir(self, fs_id, path):
+    @RESTController.Resource('DELETE', path='/{dir_path}')  # noqa: F811
+    def tree(self, fs_id, path):  # noqa: E0102
```

Comment:
noqa here because I'm not sure if it's okay to have similarly named functions like this.
Just did this to get by the pre-commit hooks so it could get reviewed.

Reply:
500 internal errors result from making requests to these endpoints

Reply:
From API tests job:
```
Error EIO: Module 'dashboard' has experienced an error and cannot handle commands: Could not infer the resource ID parameters for method tree of controller CephFSTreeController. Please specify the resource ID parameters using the RESOURCE_ID class property
2020-07-24 13:28:15,299.299 INFO:__main__:test_standby (tasks.mgr.test_dashboard.TestDashboard) ... ERROR
2020-07-24 13:28:15,300.300 INFO:__main__:Stopped test: test_standby (tasks.mgr.test_dashboard.TestDashboard) in 35.197548s
```

Reply:
> noqa here because I'm not sure if it's okay to have similarly named functions like this.
> Just did this to get by the pre-commit hooks so it could get reviewed.

Please take a look at the ```make check``` job failure:
```
mypy run-test: commands[0] | mypy --config-file=../../mypy.ini cephadm/module.py mgr_module.py dashboard/module.py prometheus/module.py mgr_util.py orchestrator/__init__.py progress/module.py rook/module.py osd_support/module.py test_orchestrator/module.py volumes/__init__.py
dashboard/plugins/feature_toggles.py:16: note: In module imported here,
dashboard/module.py:47: note: ... from here:
dashboard/controllers/cephfs.py: note: In member "tree" of class "CephFSTreeController":
dashboard/controllers/cephfs.py:409: error: Name 'tree' already defined on line 399
dashboard/controllers/cephfs.py: note: In member "quota" of class "CephFSQuotaController":
dashboard/controllers/cephfs.py:436: error: Name 'quota' already defined on line 424
dashboard/controllers/cephfs.py: note: In member "snapshot" of class "CephFSSnapshotController":
dashboard/controllers/cephfs.py:469: error: Name 'snapshot' already defined on line 454
Found 3 errors in 1 file (checked 11 source files)
```